setup/scripts/xml/latency_scan_external_triggers.xml

```diff
@@ -148,18 +148,6 @@
     <xc:Application class="gem::supervisor::GEMSupervisor" id="254" instance="0" network="local">
```

Comment:
@cgalloni should probably clean these up further in the calibration suite PR (whether complete removal, condense into one dummy config with configuration options described in the comments, etc.,)

Reply:
:+1: